curriculum/challenges/english/01-responsive-web-design/basic-css-cafe-menu/part-084.md

```diff
@@ -8,9 +8,10 @@ isHidden: true
 ## Description
 <section id='description'>
 
-The image you added is not centered horizontally like the `Coffees` heading above it.  `img` elements are "like" inline elements.
+You change properties of a link when the link is actually clicked by using a <dfn>pseudo-selector</dfn> that looks like `a:active { propertyName: propertyValue }`
```

Comment:
You change properties of a link when the link is actually clicked by using a <dfn>pseudo-selector</dfn> that looks like `a:active { propertyName: propertyValue }`.